x-pack/plugins/ml/public/application/routing/routes/jobs_list.tsx

```diff
@@ -45,13 +47,12 @@ export const jobListRouteFactory = (navigateToPath: NavigateToPath, basePath: st
 });
 
 const PageWrapper: FC<PageProps> = ({ deps }) => {
```

Comment:
Looks like we can remove `deps` and remove it from PageProps here since there's no dependency.

Reply:
Removed in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75